tests/pdctl/operator/operator_test.go

```diff
@@ -162,6 +163,14 @@ func TestOperator(t *testing.T) {
 			expect: "status: RUNNING",
 			reset:  []string{"-u", pdAddr, "operator", "remove", "3"},
 		},
+		{
+			// operator add split-region <region_id> [--policy=scan|approximate|usekey] [--keys=xxx(xxx is hex encoded string)]
```

Comment:
Perhaps all similar comments above need to be changed.

Reply:
fixed